src/main/java/ch/njol/skript/classes/data/DefaultOperations.java

```diff
@@ -96,20 +97,24 @@ public class DefaultOperations {
 			long scalar = right.longValue();
 			if (scalar < 0)
 				return null;
-			return new Timespan(Math2.multiplyClamped(left.getMilliSeconds(), scalar));
+			return new Timespan(Math2.multiplyClamped(left.getAs(TimePeriod.MILLISECOND), scalar));
 		}, (left, right) -> {
 			long scalar = left.longValue();
 			if (scalar < 0)
 				return null;
-			return new Timespan(scalar * right.getMilliSeconds());
+			return new Timespan(scalar * right.getAs(TimePeriod.MILLISECOND));
 		});
 		Arithmetics.registerOperation(Operator.DIVISION, Timespan.class, Number.class, (left, right) -> {
 			long scalar = right.longValue();
 			if (scalar <= 0)
 				return null;
-			return new Timespan(left.getMilliSeconds() / scalar);
+			return new Timespan(left.getAs(TimePeriod.MILLISECOND) / scalar);
 		});
 
+		// Timespan / Timespan = Number
+		Arithmetics.registerOperation(Operator.DIVISION, Timespan.class, Timespan.class, Number.class,
+				(left, right) -> left.getAs(TimePeriod.MILLISECOND) / (double) right.getAs(TimePeriod.MILLISECOND));
```

Comment:
I think it might be better to put it under the `Timespan - Timespan` section, but it doesn't really matter

Reply:
Yeah I thought it felt different enough due to the number return type.